File: app/src/main/java/plantopia/sungshin/plantopia/LoginActivity.java
package plantopia.sungshin.plantopia;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class LoginActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
    }

    public void signInBtnOnClicked(View view) {
        Intent i = new Intent(getApplicationContext(), MainActivity.class);
        startActivity(i);
        finish();
    }
}

File: app/src/main/java/plantopia/sungshin/plantopia/LoginActivity.kt
package plantopia.sungshin.plantopia

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View

class LoginActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
    }

    fun signInBtnOnClicked(view: View) {
        val i = Intent(applicationContext, MainActivity::class.java)
        startActivity(i)
        finish()
    }
}
